Src/AmrCore/AMReX_Interp_1D_C.H

```diff
@@ -105,6 +106,35 @@ face_linear_interp_x (int i, int /*j*/, int /*k*/, int n, Array4<T> const& fine,
     }
 }
 
+template <typename T>
+AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+void ccquartic_interp (int i, int /*j*/, int /*k*/, int n,
+                       IntVect const& /* ratio */,
+                       Array4<T const> const& crse,
+                       Array4<T>       const& fine ) noexcept
+
+{
+    // Note: there are asserts in CellConservativeQuartic::interp()
+    //       to check whether ratio is all equal to 2.
+
+    const Array1D<Real, -2, 2> cL = { -0.01171875,  0.0859375, 0.5, -0.0859375, 0.01171875 };
+
+    int ic = amrex::coarsen(i,2);
+    int irx = std::abs(i % 2);
+
+    Array1D<Real, 0, 1> ftmp;
+
+    ftmp(0) = (T)2.0 * ( cL(-2)*crse(ic-2,0,0,n)
+                       + cL(-1)*crse(ic-1,0,0,n)
+                       + cL( 0)*crse(ic  ,0,0,n)
+                       + cL( 1)*crse(ic+1,0,0,n)
+                       + cL( 2)*crse(ic+2,0,0,n) );
+    ftmp(1) = (T)2.0 * crse(ic,0,0,n) - ftmp(0);
+
+    fine(i,0,0,n) = ftmp(irx);
```

Comment:
There is no reason to have an array here and compute both.  We could do something like
```
Real ftmp = .....;
if (ic*2 != i) {
  ftmp = 2.0_rt * crse(ic,0,0,,n) - ftmp;
}
fine(i,0,0,n) = ftmp;
```